src/catalog/src/builtin.rs

```diff
@@ -6997,14 +6997,13 @@ pub static MZ_CLUSTER_REPLICA_HISTORY: LazyLock<BuiltinView> = LazyLock::new(||
                     details ->> 'replica_id' AS replica_id,
                     details ->> 'replica_name' AS replica_name,
                     details ->> 'cluster_name' AS cluster_name,
+                    details ->> 'cluster_id' AS cluster_id,
                     occurred_at
                 FROM mz_catalog.mz_audit_events
                 WHERE
                     object_type = 'cluster-replica' AND event_type = 'create'
                         AND
                     details ->> 'replica_id' IS NOT NULL
-                        AND
-                    details ->> 'cluster_id' !~~ 's%'
```

Comment:
I'm using `mz_cluster_replica_history` in the console to show the size / other replica information in our graphs throughout time. Because we expose system clusters, I removed this check. However, I'm not sure if this breaks anything else downstream.